dockerfiles/run/docker-compose.yml

```diff
@@ -10,7 +10,7 @@ services:
       - rabbitmq
       - swift
     entrypoint: bash -c "java -Dworking.directory=/root/ -Dlogback.configurationFile=/root/conf/logback.xml -jar james-server.jar"
-    image: linagora/james-rabbitmq-project:latest
+    image: apache/james:distributed-3.6.0
```

Comment:
is there a doc on what the naming scheme is  ? (I created a custom SMTP only server, I wasn't too sure how to name it) if the naming scheme is clear I intend to align with it before proposing this assembly back to the project.